src/Viz.js

```diff
@@ -249,6 +250,18 @@ export default class Viz extends BaseClass {
       strokeWidth: constant(0)
     };
     this._solo = [];
+
+    this._subtitleClass = new TextBox();
+    this._subtitleConfig = {
+      ariaHidden: true,
+      fontSize: 12,
```

Comment:
I feel like the default subtitle fontSize should be smaller than the title's fontSize 🤔 but seeing that they are both currently `12`, what do you think about increasing the title's default fontSize? We actually set it to `20` in the OEC, so how about `16`?

Reply:
I was going to suggest the same haha, it seemed strange to me to have both the same size.
Now it would be title: 16, subtitle: 12 and total: 10.